pkg/security/ebpf/c/include/helpers/network/stats.h

```diff
@@ -113,6 +113,11 @@ __attribute__((always_inline)) void flush_pid_network_stats(u32 pid, void *ctx,
 }
 
 __attribute__((always_inline)) void count_pkt(struct __sk_buff *skb, struct packet_t *pkt) {
+    // only count TCP & UDP packets for now
+    if (pkt->ns_flow.flow.l4_protocol != IPPROTO_TCP && pkt->ns_flow.flow.l4_protocol != IPPROTO_UDP) {
+        return;
+    }
```

Comment:
Not sure why we don't count icmp packets here. It may be interesting to detect data leak using icmp tunnels I guess

Reply:
It is not supported by the network flow feature user space side. It generates errors if I let the packet being processed here. I'll address the network flow use case in a follow-up PR.